docs/getting-involved-with-sourcecred.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Getting Involved with SourceCred
+description: A guide for becoming a contributor in SourceCred's internal community.
+---
+If you are interested in joining the internal community of contributors who are working together to make SourceCred a reality, then this is the doc that will help you find your way. If you're looking for information about using SourceCred as a tool in a different community/project, go to \[Using SourceCred] instead.
+
+This document is less a step-by-step guide, and more an explanation of what it looks like to go from being a newcomer to a full-fledged contributor within the SourceCred community accompanied by the resources that will help you get there. Experience these phases on your own timeline; no phase of getting to know SourceCred is any better or worse than another. This is your experience, and we encourage you to engage with it to the level that fits you.
+
+So, how does one "get involved" with SourceCred? ;)
+
+\[Blush gif? ^_^]
+
+# Some General Tips:
+
+SourceCred is still young, and we don't have many automated onboarding tools yet. That means you'll do best if are willing to dive in, ask questions, explore resources, and otherwise utilize self-directed learning. However, this doesn't mean we're throwing you to the sharks in a 'sink or swim' kind of situation. What we lack in polished or automated onboarding flows, we make up for in high community engagement. This means that your best chance at getting up to speed with what's going on and how to help is to get connected to the community. Come to calls, talk to contributors with more context, ask questions, and offer your help. We'll be glad to welcome you.
```

Comment:
Took out some parts about automated onboarding tools since it's not very common to have them in the first place, make it more general
```suggestion
SourceCred is still young, and we don't have a fully fleshed out onboarding process yet. That means you'll do best if are willing to dive in, ask questions, explore resources, and otherwise utilize self-directed learning. However, this doesn't mean we're throwing you to the sharks in a 'sink or swim' kind of situation. We are a highly engaged community, so your best chance at getting up to speed with what's going on and how to help is to be active and get connected in our communication channels. Come to calls, talk to contributors with more context, ask questions, and offer your help. We'll be glad to welcome you.
```